pom.xml

```diff
@@ -687,6 +687,7 @@
           </systemProperties>
           <reuseForks>true</reuseForks>
           <forkCount>${concurrency}</forkCount>
+          <argLine>-Djava.awt.headless=true ${argLine}</argLine>
```

Comment:
AFAIR this is to allow to add options in the command line with -DargLine=xxxx without loosing this config entry